### ISSUE TITLE
Questions regarding collisions and current rubric requirements

### ISSUE THREAD

**Issue by huangan** [usc-csci201-fall2013/simcity201#87]
Hello Professor,
Regarding the issue with collisions in our city, I have a few questions on how the following requirements will be graded.
1.  Two vehicles collide.
2.  Then ... (you design it).
3.  A vehicle hits a pedestrian.
4.  Then ... (you design it)

Specific concerns for the above requirements:

If in our city, vehicles can stack on top of each other and pedestrians due to not having A*, would this requirement be docked IF we had an explicit explosion animation played when an “actual” accident occurs and runs through the actual scenario? 
1.  Vehicles should behave appropriately at intersections.
2.  Enough vehicle traffic to show stopping/starting at intersections.

Specific concerns for the above requirements:

If in our city, vehicles DO stop and start at intersections but can stack on top of each other. For example, 4 cars can be on top of each other but all stopping and starting correctly at intersections, would this requirement be docked?
Basically, does “behave appropriately” or “Enough vehicle traffic to show stopping/starting” encompass collision detection?
1.  Routing behavior must show that agents don't bump into buildings, tables, etc.

Obviously for this particular requirement, we would be docked. The ultimate question that arises is: If the other requirements mentioned in this issue do not have collision detection, would only this requirement be docked or would all of them be docked?

@dwilczyn

**Comment by dwilczyn** [comment]
We will have to see what you have during the final demonstration. Collisions are collisions and multiple cars can collide. You will have to determine if your scenario is reasonable. Your "stacking" description does not sound good; vehicles don't stack and then behave correctly.

**Comment by huangan** [comment]
Thank you for your time and answer Professor,  but I still have one more very specific question to clear up. 

It seems like you are implying that getting full credit for the requirements "Vehicles should behave appropriately at intersections," or "Enough vehicle traffic to show stopping/starting at intersections," is contingent upon my team completing a collision detection system in our city. 

If we do have cars drawn on top of each other but not "crashing" by our design's definition during these scenarios, would my team be docked for these scenarios in addition to the "Routing behavior must show that agents don't bump into buildings, tables, etc" requirement, or would we only be docked on "Routing behavior must show that agents don't bump into buildings, tables, etc"?

Thanks  again for considering my team's question, I know you have a very busy schedule.
